pkg/resource_builders/envoyconfig/clusters.go

```diff
@@ -0,0 +1,68 @@
+package envoyconfig
+
+import (
+	"time"
+
+	"github.com/3scale-ops/marin3r/pkg/envoy"
+	saasv1alpha1 "github.com/3scale/saas-operator/api/v1alpha1"
+	envoy_config_cluster_v3 "github.com/envoyproxy/go-control-plane/envoy/config/cluster/v3"
+	envoy_config_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	envoy_config_endpoint_v3 "github.com/envoyproxy/go-control-plane/envoy/config/endpoint/v3"
+	envoy_extensions_upstreams_http_v3 "github.com/envoyproxy/go-control-plane/envoy/extensions/upstreams/http/v3"
+	"google.golang.org/protobuf/types/known/anypb"
+	"google.golang.org/protobuf/types/known/durationpb"
+	"google.golang.org/protobuf/types/known/wrapperspb"
+)
+
+func Cluster_v1(desc envoyDynamicConfigDescriptor) (envoy.Resource, error) {
+	opts := desc.(*saasv1alpha1.Cluster)
+
+	cluster := &envoy_config_cluster_v3.Cluster{
+		Name:           desc.GetName(),
+		ConnectTimeout: durationpb.New(1 * time.Second),
```

Comment:
Right now we have deployed `1s` for backend, and `2s` for echo-api and apicast

So now we are harcoding a possible lower value compared to the one used on 2 cases...  are we OK with this not customizable lower value?

Reply:
It's the connection timeout, not the request timeout. It's the time we wait for the connection to the upstream to be established. 1 second is huge. There was a difference there ... because no one ever noticed that there was a different configuration between the two.

Reply:
fair enough!